public/data/processes.json

```diff
@@ -149,36 +149,6 @@
     "waste": 0.0,
     "alias": null
   },
-  {
-    "name": "Vapeur à partir de gaz naturel (mix de technologies de combustion et d'épuration des effluents gazeux|en sortie de chaudière|Puissance non spécifiée), ES",
```

Comment:
Note to reviewers, esp. incl. @paulboosz: this Spanish steam process wasn't used anywhere so I took the liberty to remove it.